README.rst

```diff
@@ -5,15 +5,19 @@ This is a minimal Python implementation of a Portier Relying Party. It is
 implemented in Python 3. To get started:
 
 ```bash
-# Optional: create a virtualenv
+# Optional: Create a virtualenv.
```

Comment:
Please put these style tweaks in a separate commit.

Also, like in the broker PR, most of these comments feel overly explicit/obvious to me.